src/core/localization/translations/en/common.json

```diff
@@ -27,7 +27,8 @@
     "focused_geometry_editor": "Draw or edit\ngeometry",
     "edit_in_osm": "Edit map in OSM",
     "tools_label": "Tools",
-    "selected_area_label": "Selected area"
+    "selected_area_label": "Selected area",
+    "upload_analysis": "Upload\nanalysis"
```

Comment:
@alesiahil need better wording - there's not just one type of analysis we can support, and it is also not really uploading anything